Sources/Target/Linux/Process.cpp

```diff
@@ -327,13 +327,19 @@ ErrorCode Process::getMemoryRegionInfo(Address const &address,
     uint64_t inode;
     int nread;
 
-    if (std::fgets(buf, sizeof(buf), fp) == nullptr)
-      break;
+    if (::fgets(buf, sizeof(buf), fp) == nullptr) {
```

Comment:
This makes it look like ability to drop `std::` is implementation-defined.

http://stackoverflow.com/questions/29897661/when-using-standard-c-functions-in-c-is-the-std-prefix-required